client/views/actions/invite.tpl

```diff
@@ -1,9 +1,9 @@
 {{> ../user_name nick=from}}
-invited
+{{translate 'client.invited.invited'}}
 {{#if invitedYou}}
-	you
+	{{translate 'client.invited.you'}}
 {{else}}
 	{{> ../user_name nick=invited}}
 {{/if}}
-to
+{{translate 'client.invited.to'}}
 {{{parse channel}}}
```

Comment:
I don't think this is sustainable. Assuming sentence order being `[nick] [invited] [nick] [to] [channel]` is likely not going to be true in a bunch of languages. For example, in French, it would work when reporting about someone else: `[bob] [a invité] [alice] [sur] [#thelounge]`, but when user gets invited, that would be: `[bob] [vous] [a invité sur] [#thelounge]`.

I barely remember anything from my German years, but I believe the structure would be something like: `[bob] [has] [you] [to] [#thelounge] [invited]` or something like that.

Reply:
I still very much don't like this, but I have not found a nice solution either.

The only thing I could come up with is having a `client.invite.userFirst = true` in the translation file, and using that here to decide if the user/'you' or the channel goes first, then using your approach and having `client.invited.prefix`, `.middle`, and `.suffix`. I don't fully like this either though, since it puts logic inside the translation file